### DISCUSSION TITLE
New Beginnings

### DISCUSSION THREAD

**Discussion by wowakan** [pwillworth/galaxyharvester#56] [General]
Hello,

Is there a way you can wipe out all the resources for New beginnings please? We did a total restart of the server and are ready to start running new.

Thank you
Wowakan

**Comment by pwillworth** [comment]
@wowakan The way to accomplish this is for you to submit a new Galaxy entry here https://galaxyharvester.net/galaxy.py/new  

Then we retire the old one

**Comment by wowakan** [comment]
Sounds good. Thank you
